### PR TITLE
SRW-96 - Implement missing events on WEB

### PR DESCRIPTION
**Pull request by cmonti-bc**
## Description (optional)
image

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/Interest/DepositForm/template.success.tsx [hunk 1]
```diff
@@ -616,6 +631,8 @@ const DepositForm: React.FC<InjectedFormProps<{ form: string }, Props> & Props>
             height='48px'
             nature='primary'
             type='submit'
+            // @ts-ignore
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/modals/Interest/DepositForm/template.success.tsx:634]: What's the ts error here?

Comment (cmonti-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/Interest/DepositForm/template.success.tsx:634]: image

Comment (cmonti-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/Interest/DepositForm/template.success.tsx:634]: I've tried with every every event type but it seems that the button props are closed, no clue why I can't add a simple `onClick`

Comment (cmonti-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/Interest/DepositForm/template.success.tsx:634]: ohh, just got it, I can add the event on the submit function. Fixing it!

**Review by blockdylanb — APPROVED** [review]

**Pull request merged**